tools/gyp/pylib/gyp/generator/make.py

```diff
@@ -211,10 +211,10 @@ def CalculateGeneratorInputInfo(params):
 
 LINK_COMMANDS_AIX = """\
 quiet_cmd_alink = AR($(TOOLSET)) $@
-cmd_alink = rm -f $@ && $(AR.$(TOOLSET)) crs $@ $(filter %.o,$^)
+cmd_alink = rm -f $@ && $(AR.$(TOOLSET)) -Xany crs $@ $(filter %.o,$^)
 
 quiet_cmd_alink_thin = AR($(TOOLSET)) $@
-cmd_alink_thin = rm -f $@ && $(AR.$(TOOLSET)) crs $@ $(filter %.o,$^)
+cmd_alink_thin = rm -f $@ && $(AR.$(TOOLSET)) -Xany crsT $@ $(filter %.o,$^)
```

Comment:
I left out changes in other dependencies until we worked on the upstream but this one is required for compilation where as the others only resulted in test failures.  Should I remove it from the patch until the upstream process is complete ?